lectures/equalizing_difference.md

```diff
@@ -206,34 +208,34 @@ prominently including $\gamma_h, \gamma_c, R$.
 
 Now let's write some Python code to compute $\phi$ and plot it as a function of some of its determinants.
 
-
 ```{code-cell} ipython3
-class equalizing_diff:
-    """
-    A class of the equalizing difference model
-    """
+# Define the namedtuple for the equalizing difference model
+EqDiffModel = namedtuple('EqDiffModel', 'R T γ_h γ_c w_h0 D π')
+
+def create_edm(R=1.05,   # Gross rate of return
+               T=40,     # Time horizon
+               γ_h=1.01, # High-school wage growth
+               γ_c=1.01, # College wage growth
+               w_h0=1,   # Initial wage (high school)
+               D=10,     # Cost for college
+               π=None):
+    
+    return EqDiffModel(R, T, γ_h, γ_c, w_h0, D, π)
+
+def compute_gap(model):
+    R, T, γ_h, γ_c, w_h0, D, π = model
+    
+    A_h = (1 - (γ_h/R)**(T+1)) / (1 - γ_h/R)
+    A_c = (1 - (γ_c/R)**(T-3)) / (1 - γ_c/R) * (γ_c/R)**4
     
-    def __init__(self, R, T, γ_h, γ_c, w_h0, D=0, π=None):
-        # one switches to the weak model by setting π
-        self.R, self.γ_h, self.γ_c, self.w_h0, self.D = R, γ_h, γ_c, w_h0, D
-        self.T, self.π = T, π
+    # Tweaked model
```

Comment:
After `π=None` put `# Extra parameter (modified version)` and then replace `Tweaked model` with `Modification for reinterpreted model`

Reply:
Hi @jstac,

Since $\pi$ is only used for the entrepreneur model, would it be a good idea to remove it here and create a new `namedtuple` for the entrepreneur model when moving it into the exercise?

Reply:
Hi @HumphreyYang , yes, good call.  Let's do exactly that.